packages/gatsby-core-utils/src/__tests__/path.js

```diff
@@ -21,4 +21,11 @@ describe(`paths`, () => {
       })
     }
   })
+  describe(`isNodePath`, () => {
+    it(`Matches common node files`, () => {
+      expect(isNodePath(`console.js`)).toBe(true)
+      expect(isNodePath(`http.js`)).toBe(true)
+      expect(isNodePath(`internal/foo`)).toBe(true)
```

Comment:
Could we add at least one test path that we expect *not* to be `true`? Something like `/Users/username/dev/project-1/node_modules/packaga-name/index.js` maybe

Reply:
> That was very quick!
> 
> We named this poorly in our internals (`nodePaths`) but it was kinda acceptable, given that it wasn't exported anywhere. Given that we are making this more public API we should take more care in naming this - I'd suggest something like `IsNodeInternalPath` or `IsNodeInternalModulePath` to more clearly convey what this function check

Thanks! I was struggling a bit with the naming. I'll change it to `isNodeInternalModulePath`

Reply:
> Could we add at least one test path that we expect _not_ to be `true`? Something like `/Users/username/dev/project-1/node_modules/packaga-name/index.js` maybe

Added an unhappy path to the unit test.